image-layout.md

```diff
@@ -185,7 +178,7 @@ Those tags will often be represented in an image-layout repository with matching
       }
     },
     {
-      "mediaType": "application/xml",
```

Comment:
this example is illustrating that other mediatypes can be included that are not just OCI mediatyps